src/index.css

```diff
@@ -10,4 +10,9 @@ input {
   border: 2px solid #ffffff;
   text-align: center;
   font-size: 16px;
+}
+
+.align-link {
```

Comment:
would prefer this to be a styled component

maybe try this?
https://www.styled-components.com/docs/basics#attaching-additional-props